config/development/images.yaml

```diff
@@ -6,26 +6,10 @@ spec:
   template:
     spec:
       initContainers:
-        # Install client libraries for unreleased versions of FDB
```

Comment:
I believe a block like this is still necessary, because the `foundationdb-kubernetes-init-7-1-primary` block doesn't install any of the CLIs.

Reply:
I think we can get rid of all the custom changes here by just adding the init container to the opreator in the [manager.yml](https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/config/deployment/manager.yaml#L32-L48).